plugins/ledmatrix-flights/manager.py

```
@@ -2255,7 +2321,36 @@ def update(self) -> None:
+                self._pirep_data = {
+                    icao: self._metar_fetcher.fetch_pireps(icao, self.metar_pirep_distance_nm)
+                    for icao in self.metar_airports
+                }
+            else:
+                self._pirep_data = {}
+            self._sigmet_data = (
+                self._metar_fetcher.fetch_sigmets() if self.metar_show_sigmet else []
+            )
+            self.logger.info(
+                f"[Flight Tracker] Refreshed airport weather for {list(self._metar_data.keys())}")
+        except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)